<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Arial&quot;;
color: #006bae;</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Arial&quot;;
color: #006bae;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>199</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background: white; 
	border: 2px solid  #006bae;
	border-radius: 10px;
	font-family: Arial;
	color: rgb(0, 0, 0);
	font: 10pt;
}

QLineEdit:focus {
	border: 2px solid rgb(0, 67, 109) ;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background: white; 
	border: 2px solid  #006bae;
	border-radius: 10px;
	font-family: Arial;
	color: rgb(0, 0, 0);
	font: 10pt;
}

QLineEdit:focus {
	border: 2px solid rgb(0, 67, 109) ;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 10pt &quot;Arial&quot;;
	background-color: #006bae;
	color: white;
}


QPushButton:hover{
	background-color: rgb(0, 84, 136);

}

QPushButton:pressed{
	border: 2px solid  rgb(0, 107, 174);

}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Arial&quot;;
color: white;
background-color: rgb(0, 107, 174);</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
